src/org/traccar/protocol/EelinkProtocolDecoder.java

```diff
@@ -112,9 +118,7 @@ private String decodeAlarm(Short value) {
     }
 
     private void decodeStatus(Position position, int status) {
-        /**
-         * 16 bits, read lowest 0 to 15
-         */
+        /* 16 bits, read lowest 0 to 15 */
```

Comment:
I don't understand the purpose of this comment. What value does it provide? Also, as I said, don't use multi-line comments (`/* */`).